README.md

```diff
@@ -12,9 +12,9 @@ Run the following commands on an all-snap image:
 
     $ sudo snap install --beta --devmode classic
     $ sudo classic
-    (classic)ubuntu@localhost:~$ sudo apt update
```

Comment:
ah, well, let me re-add it then, i meant to show that you dont need to run it anymore ...